### PR TITLE
#14 - SSL toggle for the database

### PR DESCRIPTION
**Pull request by denis-onder**
Issue #14 

TL;DR - What does this implement?
---------------------------------
Satisfies the requirements of #14, implements a simple way to toggle between database connections.

How to Test
-----------
Set the new environmental variables and try starting the server.

Checklist
---------

 - [X] Do we need to test this?
 - [ ] Were tests written?
 - [x] Does this meet coding standards?
 - [x] Is this is code change?
 - [ ] Is this a meta change (e.g issue template)?
 - [x] Is this pull request ready for review?
 - [ ] Is this a breaking change (e.g not backwards compatible with clients)?
 
ESLint Output
-------------

```
Warning: React version was set to "detect" in eslint-plugin-react settings, but the "react" package is not installed. Assuming latest React version for linting.
```

Reviewers
---------

@gmemstr

### CODE REVIEW

File: app/config/index.js [hunk 1]
```diff
@@ -8,12 +10,26 @@ module.exports = {
...
+    staging: {
+      name: process.env.STAGING_DB_NAME,
+      username: process.env.STAGING_DB_USER,
+      password: process.env.STAGING_DB_PASSWORD,
+      host: process.env.STAGING_DB_HOST,
+      dialect: process.env.STAGING_DB_DIALECT || 'postgres',
+      port: process.env.STAGING_DB_PORT || 5432,
+      dialectOptions: {
+        ssl: true,
+        ca: readFileSync(resolve(__dirname, '../keys', 'certificate.crt')),
```
Comment (gmemstr) [inline app/config/index.js]: Not strictly required, but is nice to have I suppose. Just have to document where the key should be. Also, the SSL should toggle based on `server.environment === "staging"` (or something)

File: app/keys/certificate.crt [hunk 2]
```diff
@@ -0,0 +1,25 @@
+-----BEGIN CERTIFICATE-----
```
Comment (gmemstr) [inline app/keys/certificate.crt]: Let's not commit the cert, since this will differ when we deploy different clusters.
